packages/flex-plugin-scripts/src/prints/validateSuccessful.ts

```diff
@@ -35,7 +35,7 @@ const printWarnings = (issues: Warning[]): void => {
         }
         if (link) {
           logger.newline();
-          logger.info(`More details ${logger.coloredStrings.bold(logger.linkText('here', link))}`);
+          logger.info(`More details ${logger.coloredStrings.link(logger.coloredStrings.underline(link))}`);
```

Comment:
Can we make the text something like this -> `More details here - https://example.com`

Reply:
here does not nice at this place, something like `For more details, visit: https://example.com/` will work ?

Reply:
Yup, better 👍🏼